src/components/Home.js

```diff
@@ -16,7 +17,10 @@ const Home = ()=>{
   const [homeData, setHomeData] = useState(initialHomeData)
  
   useEffect(()=>{
-//To-Do: find out how to login and get user after logging in... where does userID live
+//No longer think I need this function because it occurs at login, but what if I renavigate to the home page?
```

Comment:
Could you perhaps store the token in memory ( `useState` ) once it has been received? Is the token needed to navigate to the home page?

Reply:
I am thinking about making the login page separate from the home page. Using the protectedroute to pass the user to the home page. The reason being is this will allow the token to automatically expire. and log the user out.

Reply:
That sounds like a good idea.